ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php

```diff
@@ -773,7 +774,8 @@ protected function formatViewValue($key, $rawValue) {
         break;
 
       case 'Money':
-        $formatted = \CRM_Utils_Money::format($rawValue);
+        $currencyField = $this->getCurrencyField($key);
+        $formatted = \CRM_Utils_Money::format($rawValue, $data[$currencyField] ?? NULL);
```

Comment:
We should switch this to `\Civi::format()->money()`

Reply:
Ok @eileenmcnaughton I've made that change - it altered the formatting but all improvements I think... I've updated the screenshot. Adding CA in front of the canadian dollar sign is a good touch. I'm not sure why it removed the trailing `.00` from Japanese currency but seems fine.

Reply:
I don't think Japanese currency has trailing bits....